codegenerator/cli/templates/static/codegen/src/eventFetching/FetchState.res

```
@@ -373,6 +365,10 @@ let registerDynamicContracts = (
           blockNumber: Pervasives.max(startBlockKey->Int.fromString->Option.getExn - 1, 0),
           blockTimestamp: 0,
         },
+        // FIXME: Can the normalSelection be empty?
+        // Probably only on pre-registration, but we don't
+        // register dynamic contracts during it
+        ~selection=fetchState.normalSelection,
```

Lets throw an error if it's empty